lib/routes/szftedu/dongtai.ts

```diff
@@ -0,0 +1,65 @@
+import { Route } from '@/types';
```

Comment:
You're submitting PR for namespace `yamap`. Please remove files related to other namespace, i.e., `szftedu` from the PR.